### PR TITLE
Add Tealium module

### PR DESCRIPTION
**Pull request by ianhampton**
Basic Tealium module to handle page views and events.

### CODE REVIEW

**Comment by 0xR** [comment]
Maybe you can make it possible to load tealium from an angular module. It could have some settings which can be set from a config block. This would make it easier to have some logic determine which tealium envorinment to use while keeping this logic under an angular unit test.

File: src/angulartics-tealium.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+angular.module('angulartics.tealium', ['angulartics'])
+.config(['$analyticsProvider', function($analyticsProvider){
+  var dataCheck;
+  
+  /**
+  * dataCheck function to convert data sources into Tealium proposed standard. i.e. 'page_name'.
+  */
+
+  dataCheck = function(data){
```
Comment (lunks) [inline src/angulartics-tealium.js]: convertHashToSnakeCase or something similar looks like a better name. What do you think?

Comment (ianhampton) [inline src/angulartics-tealium.js]: Agreed, that would make more sense.

**Comment by ianhampton** [comment]
@lunks agreed, function name updated.

**Comment by luisfarzati** [comment]
We are in the process of moving all plugins off the core and into their own repo (see #377), so PRs for plugins are no longer accepted. Instead, I'd appreciate if you create a repo for this plugin on your side and make sure to comply with the [plugin contribution rules](https://github.com/luisfarzati/angulartics/wiki/Plugin-contribution-rules).

Once you have everything ready, let us know and if everything's right we can adopt the plugin into the [Angulartics org](https://github.com/angulartics).

Thank you!

**Pull request closed without merging**